eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -38,7 +38,7 @@ stages:
                   inputs:
                     filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
                     arguments: >
-                      -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
+                      -artifactLocation '$(PIPELINE.WORKSPACE)\packages'
```

Comment:
By the way this path will only work on windows. Why did the other paths not work?

Reply:
Turns out the culprit was the quotes https://github.com/Azure/azure-sdk-for-net/pull/28407